x/chainmain/client/cli/query.go

```diff
@@ -35,6 +37,52 @@ func GetQueryCmd() *cobra.Command {
 	return chainmainQueryCmd
 }
 
+func QueryAllTxCmd() *cobra.Command {
+	// returns a command to search through transactions by address.
+	cmd := &cobra.Command{
+		Use:   "txs-all [address]",
+		Short: "Query for all  paginated transactions by address",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, queryErr := client.GetClientQueryContext(cmd)
+			if queryErr != nil {
+				return queryErr
+			}
+			addrStr := args[0]
+			_, addressErr := sdk.AccAddressFromBech32(addrStr)
+			if addressErr != nil {
+				return addressErr
+			}
+
+			var tmEvents []string
+			events := [...]string{"message.sender", "transfer.recipient", "withdraw_rewards.validator", "ibc_transfer.receiver"}
```

Comment:
not sure `ibc_transfer.sender` is also valid?

Reply:
I'll test